hcipy/aperture/generic.py

```diff
@@ -362,6 +362,23 @@ def make_rotated_aperture(aperture, angle):
 	'''
 	return lambda grid: Field(aperture(grid.rotated(-angle)), grid)
 
+def make_shifted_aperture(aperture, shift):
+	'''Create a rotated version of `aperture`.
```

Comment:
Update docstring. This is just a copy-pasted version from some other function.

Reply:
Done. I have also added a test for this function.